becquerel/core/energycal.py

```diff
@@ -332,13 +347,21 @@ def valid_coeffs(self):
     def slope(self):
         """Return the slope coefficient value."""
 
-        return self._coeffs['b']
+        try:
+            return self._coeffs['b']
+        except KeyError:
```

Comment:
This could also be done with:

```python
if 'b' in self._coeffs:
    return self._coeffs['b']
else:
    raise EnergyCalError('Slope coefficient not yet supplied or calculated.')
```

But I think the `KeyError` catch is more elegant

Reply:
i agree with using `KeyError`